circle.yml

```diff
@@ -6,6 +6,7 @@ machine:
     PARALLEL_TEST_PROCESSORS: 8
 
     PATH: $HOME/.cabal/bin/:$PATH
+    GO_PROJECT: "$HOME/.go_workspace/src/github.com/Originate/gt"
```

Comment:
The typical pattern is that the top-level repo is a library, then you'd have a `cmd/gt/main.go` that imports `github.com/Originate/git-town` and handles argument parsing and whatnot and that's the binary you ship